v22.2/create-external-connection.md

```diff
@@ -0,0 +1,170 @@
+{{site.data.alerts.end}}
+
+## External connection URI format
+
+To form the external connection URI in operation statements, use the `external://` scheme followed by the name of the external connection. 
+
+For an external connection named `backup_storage`:
+
+~~~
+'external://backup_storage'
+~~~
+
+See the [examples](#create-an-external-connection-for-cloud-storage) in the next section for details on external connection creation.
+
+## Examples
+
+The examples in this section demonstrate a couple of the storage and operation options that external connections support.
+
+## Create an external connection for cloud storage
+
+In this example, you create an external connection for an Amazon S3 bucket that will store your backups. Then, you use the external connection to restore the backup to your cluster.
+
+1. Define your external connection that references the S3 bucket's URI:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE EXTERNAL CONNECTION backup_bucket AS 's3://bucket name?AWS_ACCESS_KEY_ID={access key}&AWS_SECRET_ACCESS_KEY={secret access key}';
+    ~~~
+
+1. To check a list of all external connections, run:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~sql
+    SHOW CREATE ALL EXTERNAL CONNECTIONS;
+    ~~~
+
+    ~~~
+    connection_name |                                                                              create_statement
+    ----------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+    backup_bucket   | CREATE EXTERNAL CONNECTION 'backup_bucket' AS 's3://bucket name?AWS_ACCESS_KEY_ID={access key}&AWS_SECRET_ACCESS_KEY={secret access key}'
+    ~~~
+
+1. Run the backup to your S3 bucket using the external connection's name:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~sql
+    BACKUP DATABASE movr INTO 'external://backup_bucket' AS OF SYSTEM TIME '-10s' WITH revision_history;
+    ~~~
+
+1. Use [`SHOW BACKUP`](show-backup.html) to view your backups in the storage defined by an external connection:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~sql
+    SHOW BACKUPS IN 'external://backup_bucket';
+    ~~~
+    ~~~
+            path
+    ------------------------
+    2022/09/19-134559.68
+    2022/10/10-192044.40
+    (2 rows)
+    ~~~
+
+1. In the event that a restore is necessary, use `RESTORE` with the external connection: 
+
+    {% include_cached copy-clipboard.html %}
+    ~~~sql
+    RESTORE DATABASE movr FROM LATEST IN 'external://backup_bucket';
+    ~~~
+
+1. To delete the external connection:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~sql
+    DROP EXTERNAL CONNECTION backup_bucket;
+    ~~~
+
+## Create an external connection for a changefeed sink
```

Comment:
Nice! Since kafka is the only sink target with unique requirements (per blue note), it's great that it gets its own example to help showcase them, love it!